### PR TITLE
fix a few minor bugs in ppo.py

### PR DESCRIPTION
**Pull request by kykim0**
# What does this PR do?
Fixes several minor bugs in the ppo.py example including the incorrect argument in build_dataset, missing argument in ScriptsArguments.

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/trl/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a GitHub issue? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/trl/tree/main/docs).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: examples/scripts/ppo.py [hunk 1]
```diff
@@ -174,7 +175,7 @@ def collator(data):
     "max_new_tokens": 32,
 }
 
-for _epoch, batch in tqdm(enumerate(ppo_trainer.dataloader)):
+for _epoch, batch in tqdm(enumerate(ppo_trainer.dataloader), total=len(ppo_trainer.dataloader)):
```
Comment (qgallouedec) [inline examples/scripts/ppo.py:178]: `total` is set to `len(iterable)` by default, why do you need this?

**Comment by qgallouedec** [comment]
Thanks for fixing this issue @kykim0! Can you just answer my remark and we're ready to merge

File: examples/scripts/ppo.py (hunk 1 above)
Comment (kykim0) [reply examples/scripts/ppo.py:177]: Thanks for the review. When I ran the original code, I didn't see the total number of iterations getting printed. My understanding is that `enumerate` returns a generator in which case `len()` wouldn't work. Alternatively, we can probably just do `tqdm(ppo_trainer.dataloader)`.

**Review by qgallouedec — APPROVED** [review]

File: examples/scripts/ppo.py (hunk 1 above)
Comment (kykim0) [reply examples/scripts/ppo.py]: Sounds good, thanks for the patch

**Comment by qgallouedec** [comment]
LGTM thanks @kykim0

**Pull request merged**